libtcmu_common.h

```diff
@@ -75,6 +77,9 @@ int tcmu_emulate_read_capacity_16(uint64_t num_lbas, uint32_t block_size, uint8_
 int tcmu_emulate_mode_sense(uint8_t *cdb, struct iovec *iovec, size_t iov_cnt, uint8_t *sense);
 int tcmu_emulate_mode_select(uint8_t *cdb, struct iovec *iovec, size_t iov_cnt, uint8_t *sense);
 
+void tcmulib_track_aio_request_start(struct tcmu_device *dev);
+void tcmulib_track_aio_request_finish(struct tcmu_device *dev, int *is_idle);
```

Comment:
I think we misunderstood each other for part of the comment/question.

I was saying in the current patches they are only used in the aio_schedule -> io_work_queue code path in your patches. We agree on that use case.

In my original comment I was not sure if you meant to only use them there or it was going to be used in the async rbd. I see your comment you are going to use them in rbd.

Were you also going to use them in file_optical and file_example's async path in a similar way you were going to use them in your rbd patches?